tools/misbehaving-jmx-server/src/main/java/org/datadog/misbehavingjmxserver/App.java

```diff
@@ -83,6 +83,9 @@ public static void main( String[] args ) throws IOException, MalformedObjectName
         // I don't think this call is actually important for jmx, the below 'env' param to JMXConnectorServerFactory is the important one
         RMISocketFactory.setSocketFactory(customRMISocketFactory);
 
+        // Explicitly set rmi server host, seems to be issues with the default on our corp computers host MacOS
```

Comment:
```suggestion
        // Explicitly set RMI hostname to specified argument value
```

I don't think its necessary to mention the specific case we're working around.  Really this way is the most correct and it should have been written this way in the first place. So you're just correcting my mistake :)